winml/lib/Api/ImageFeatureValue.cpp

```diff
@@ -155,8 +155,8 @@ BitmapBounds ImageFeatureValue::CenterAndCropBounds(
   // TODO: Do we allow smaller images?
   WINML_THROW_HR_IF_FALSE_MSG(
       WINML_ERR_INVALID_BINDING,
-      (bounds.X >= 0 && bounds.X <= m_widths[idx]) &&
-          (bounds.Y >= 0 && bounds.Y <= m_heights[idx]),
+      (bounds.X >= 0 && bounds.Width <= m_widths[idx]) &&
```

Comment:
Shouldn't it be `bounds.X + bounds.Width`?

Reply:
Actually, this condition should never fail. I'll remove this check.

---
In reply to: [379648790](https://github.com/microsoft/onnxruntime/pull/3025#discussion_r379648790) [](ancestors = 379648790)